tests/framework/test_adaptive_det.xml

```diff
@@ -9,6 +9,9 @@
     <description>
         This  test is aimed to test the capability of RAVEN to employ the Adaptive Dynamic Event Tree Sampling strategy (using RELAP7 as system code) in a larger case
     </description>
+    <revisions>
+        <revision author="alfoa" date="2019-03-07">Modified syntax in Functions as for issue #934</revision>
+    </revisions>
```

Comment:
It looks like this file wasn't revised after all? Or was it in an external XML somewhere?

Reply:
Fixed...in reality this test is never run since no RELAP7 is available anymore. 
It needs to be removed as issue #757